app/views/hiring_staff/vacancies/documents/index.haml

```diff
@@ -0,0 +1,17 @@
+.create-title
+  %h1.govuk-heading-m Create a job listing for Bexley Heath Academy
+  %h5.govuk-heading-m Step 2 of 3
+  %main.govuk-main-wrapper
+    .govuk-grid-row
+      .govuk-grid-column-two-thirds
+        %h2.govuk-heading-l Supporting documents
+        %form{:action => "#", :method => "post"}
```

Comment:
Can you break just the form out into a haml partial, please?  Chris' templates are already in `develop`, and we should use those as the source of truth to avoid merge conflicts.   You could just swap this as a partial for the dummy form he's got in there already. 

NOTE: They need some DRYing out, but they're all already there.

Reply:
Sorry for not seeing this. 
Chris templates are not in the folder I was expecting.

The controller `HiringStaff::Vacancies::DocumentsController` follows the same namespacing for where it's located in the app's directory. (Controllers -> Hiring Staff -> Vacancies -> DocumentsController)

We assumed the template would be located in a similar way. e.g. Views -> Hiring_Staff -> Vacancies -> Documents -> Index, which is why we created it there.